include/rocksdb/compaction_filter.h

```diff
@@ -13,24 +13,16 @@
 #include <string>
 #include <vector>
 
+#include "rocksdb/listener.h"
 #include "rocksdb/rocksdb_namespace.h"
 
 namespace ROCKSDB_NAMESPACE {
 
 class Slice;
 class SliceTransform;
 
-// Context information of a compaction run
-struct CompactionFilterContext {
-  // Does this compaction run include all data files
-  bool is_full_compaction;
-  // Is this compaction requested by the client (true),
-  // or is it occurring as an automatic compaction process
-  bool is_manual_compaction;
-};
```

Comment:
Just curious on if removing this struct will cause any compatibility issues with existing deployments.  It is in the public API...

Reply:
Maybe. It's unused in RocksDB for a long time though. It looks like removing this was forgotten in 9803e0d813104112f05e2dcea26058cb7cf1e408 (2015). I can add a release note.

Reply:
> add a release note

Done.